### PR TITLE
Assignment tables

### PR DESCRIPTION
**Pull request by gregmacfarlane**
I've created a demonstration template that does most of the tables and plots from the Asheville assignment comparison memo. See if you have problems building this memo, and if there are things you want changed.

### CODE REVIEW

File: inst/rmarkdown/templates/assignment_validation/skeleton.Rmd [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+table_rmse(links, volume_var, count_var, group_field = volume_var) %>%
+  mutate(`Target` = c(120, 45, 35, 30, 40)) %>%
+  kable(digits = 2, caption = "RMSE by Volume Class")
+
+table_rmse(links, volume_var, count_var, group_field = "area_type") %>%
+  kable(digits = 2, caption = "RMSE by Area Type")
+```
+
+```{r flows}
+table_flow(links, volume_var, count_var, group_field = "facility_group") %>%
```
Comment (dkyleward) [inline inst/rmarkdown/templates/assignment_validation/skeleton.Rmd]: @gregmacfarlane 
When I try to knit this file, I get the following error message:
error

**Comment by dkward2** [comment]
I commented in the file itself.  Let me know if that is the best way to provide feedback, or if it should be done here.  Does it stay associated with this pull request?

**Comment by gregmacfarlane** [comment]
No, I see it. I wonder if your `dplyr` is outdated. Try `devtools::install_github("hadley/dplyr")` and run again.

**Comment by gregmacfarlane** [comment]
I'm running `dplyr_0.4.3.9001`, btw.

**Comment by gregmacfarlane** [comment]
Actually, I think this was somewhat easy to protect against, so I'm protecting against it. But we definitely need to specify a minimum `dplyr` version.

**Comment by dkyleward** [comment]
I have updated dplyr to `0.4.3.9001`.  I'm now getting a different error.
error

btw, do you know how to copy/paste those error messages from rstudio?  I'd like to not have to upload screenshots...

File: inst/rmarkdown/templates/assignment_validation/skeleton.Rmd (hunk 1 above)
Comment (dkyleward) [inline inst/rmarkdown/templates/assignment_validation/skeleton.Rmd]: Just a dplyr version problem on my end

**Comment by gregmacfarlane** [comment]
If you build the document inside the R console then it won't print to that screen.

``` r
rmarkdown::render("inst/rmarkdown/templates/assignment_validation/skeleton.Rmd")
```

**Comment by gregmacfarlane** [comment]
I wonder if it's related to https://github.com/hadley/dtplyr/issues/7

**Comment by gregmacfarlane** [comment]
Try reversing commit 93060588ced9e461504c7867041b78a5d3c72985 and see if it works now.

**Comment by dkyleward** [comment]
Same error

**Comment by dkyleward** [comment]
One oddity is that I see uncommitted changes in my branch to `assignment_tables.R`, but I haven't modified any code.
changes

**Comment by gregmacfarlane** [comment]
It seems that you reversed the commit, which modifies the code.

**Comment by dkyleward** [comment]
Oh I see.  I actually undid the reverse by resetting my branch, but I used a mixed reset instead of hard, so the index still sees the reverse as uncommitted changes.  I'll just discard them.

**Comment by dkyleward** [comment]
My dplyr issues building the package examples appear to be entirely local (hadley/dplyr#1591).  With Travis building it successfully, feel free to merge when you're ready.

**Pull request merged**